README.md

```diff
@@ -1 +1,129 @@
+- Unluckily it won't work on old devices, this is a limitation of DroidScript and not from WashingBot itself, but nevertheless it's a pity because it's a great use for old phones :(
+
+### Device testing list
+
+WashingBot has been tested on the following devices with these results:
+
+| Device | Results | Comments |
+|--------|:---------:|----------|
+|LG Nexus 4 | :x: | Accelerometer reports garbage when screen is off.
+|Samsung Galaxy S4 | :white_check_mark: ||
+
+If you test it, just tell us and we'll add your device to the list ;)
+  
+### Future improvements & ideas
+
+- Improve UI (graphics, etc.).
+- Publish APK to Google Play (need to buy the ApkBuilder plugin).
+- Configuration editor (include for APK).
+- Ability to add/choose different notification mechanisms.
+- Ability to calibrate accelerometer (or define a threshold, etc) (PRs are welcome!).
+- Maybe decoupling the sensor from the monitor.
+- Maybe separating the sensor into another project.
+- Add DroidScript SPK to the repository, so people can easily try it.
+- See if there is a way of generating the SPK from the computer so we can include it in the toolchain. 
+- Add jsLint, etc. to the toolchain.
+- Publish graphing app for sensor data (a gist would be enough for now).
+- Testing on more devices and washing machines!
+- Maybe explain the UI here? :P
+- Check what happens if the device doesn't have accelerometer but runs DroidScript. Is that possible?
+- Detail and publish the workflow and toolchain used for developing.
+
+### Troubleshooting :feelsgood:
+
+- **Washing start is never detected:** you could do some tests by shaking you phone, does it work? Maybe you have an ultra-quiet washing machine? The bot doesn't have any low level threshold, it understands "movement" from the most minimal accelerometer readings, so maybe your device is not sensitive enough? Maybe you could try placing the phone over different areas of the washing machine and see how it goes.
+- **Washing end is never detected:** make sure you have waited long enough for the threshold to be reached (`washingThresholdMinutes` in config.js), default is 10 minutes. You should also make sure that your phone's accelerometer doesn't report garbage when the screen goes off, you can use the UDP debugging for this (or the DroidScript IDE directly also).
+- **The cycle ends at lightning speed:** make sure `useSecondsInsteadOfMinutes` is disabled in the config :P
+- **Never connects to Slack:** make sure it's properly configured, see "Slack Integration" section.
+
+### Bugs, contact & contributions :shipit:
+
+- **Got a crazy idea/problem, fixed/found a bug or included a nice comment?** Just file an issue on GitHub or send a PR!!
+- **Have free time?** You can pick some ideas from the "Future improvements & ideas" section ;)  
+Idea: File an issue with the topic you want to work on before starting, just to coordinate (is this the right way to do it?).
+- **Just wanna contact us?** lucas.devescovi :recycle: gmail.com 
```

Comment:
I didn't include your email, you can complete the Slack stuff, your email, and send another PR ;)